packages/iris-grid/src/IrisGridRenderer.js

```diff
@@ -71,6 +82,7 @@ class IrisGridRenderer extends GridRenderer {
     super.drawCanvas(state);
 
     this.drawScrim(state);
+    this.drawCellOverflowButton(state);
```

Comment:
This should be drawn before the scrim - the scrim is drawn overtop when the grid is inactive/we shouldn't be able to click on anything.